app/src/main/res/values/strings.xml

```diff
@@ -312,13 +314,13 @@
         <item>Every 4 hours</item>
         <item>Once a day</item>
     </string-array>
-    <string name="settings_sync_wifi_only">Sync over WiFi only</string>
-    <string name="settings_sync_wifi_only_on">Synchronization is restricted to WiFi connections</string>
+    <string name="settings_sync_wifi_only">Only sync over Wi-Fi</string>
```

Comment:
WiFi, not Wi-Fi. I don't care that the latter is more correct, people more commonly use WiFi.

Reply:
The actual Android settings say "Wi-Fi". There is no "WiFi".
https://en.wikipedia.org/wiki/Wi-Fi

Reply:
OKie, Wi-Fi is fine then.